pythonforandroid/recipes/sdl2_mixer/__init__.py

```diff
@@ -6,6 +11,34 @@ class LibSDL2Mixer(BootstrapNDKRecipe):
     url = 'https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-{version}.tar.gz'
     dir_name = 'SDL2_mixer'
 
-    patches = ['toggle_modplug_mikmod_smpeg_ogg.patch']
+    patches = ['toggle_modplug_mikmod_ogg.patch']
+
+    def _external_dir(self):
```

Comment:
Why do you split it into 4 methods? I would be more clear if you put it right in prebuild_arch()